CONTRIBUTING.md

```diff
@@ -2,15 +2,34 @@
 
 If you want to contribute to this repository - thank you! Before you start, have a look at the existing documentation to get an idea of the structure and writing conventions n8n uses. In writing your documentation, please follow the guidelines described below, to ensure quality and consistency with n8n styles.
 
-## Before you start
+## Docs issues only
 
-### Style
+This repo is for the documentation site.
+
+- Need help? [Ask on the forum](https://community.n8n.io/c/questions/12).
+- Got a feature request or product idea? [Propose it on the forum](https://community.n8n.io/c/feature-requests/5).
+- Found a bug in the product? [Product repo](https://github.com/n8n-io/n8n)
+
+
+## Why we might reject your contribution
+
+Due to an increase in spam and poor quality contributions, we've introduced new criteria and processes. 
+
+| Label | Action | Notes |
+| ----- | ------ | ----- |
+| `spam` | Close PR/issue; account blocked | This label is applied to things like empty issues. |
+| `ai-slop` |	Close PR/issue; account blocked | This label is applied when content is bad, wrong, and looks like it's probably the result of using AI. n8n isn't banning use of AI in contributions, so long as the contributions are high quality. |
+| `low-effort-bad` |	Close PR/issue; comment with contribution guidance; a second `low-effort-bad` leads to a block. | To avoid this, test your work to make sure it's accurate; follow the [style guide](https://github.com/n8n-io/n8n-docs/wiki/Styles); write meaningful titles and descriptions on your PR/issue. |
+| `duplicate` |	Close PR/issue; comment with contribution guidance; a second `duplicate` leads to a block. | To avoid this, check existing issues and PRs |
+
```

Comment:
I think these make sense. Just to clarify, do we want the `low-effort-bad` and `duplicate` "strikes" to be a single pool or separate?